src/forum-tool-selector-settings/ForumToolSelectorSettings.jsx

```diff
@@ -0,0 +1,150 @@
+import React, { useState } from 'react';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import StudioApiService from '../data/services/StudioApiService';
+
+var selectedForumTool; // eslint-disable-line no-unused-vars
+var setSelectedFunctions = [];
+
+function resetSelected() {
+  for (var i = 0; i < setSelectedFunctions.length; i++) {
+    setSelectedFunctions[i](false);
+  }
+}
+
+function ForumCard(props) {
+  const [hovered, setHovered] = useState(false);
```

Comment:
It is entirely possible to accomplish what you're trying to do here using CSS, please don't use JavaScript for this.